Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/workbench/RetrieveProjectManager.java

```diff
@@ -56,6 +56,18 @@
 	}
 	
 	/**
+	 * 
+	 */
+	@Override
+	@Transactional
+	public List<IProject> getProjectListByAccessibility(String accessibility) throws QuadrigaStorageException
+	{
+		List<IProject> projectList;
+		projectList =  projectShallowMapper.getProjectList(accessibility);
+		return projectList;

```

Comment:
Also, I'd do that the other way around. Let your manager class get the DTOs and then map them using the mapper. Then you can get rid of the getProjectListByAccesibility method in the mapper classes.